debian/prebuild.sh

```diff
@@ -0,0 +1,6 @@
+curl -LsSf https://www.tarantool.io/release/1.10/installer.sh | sudo bash
+
+# Workaround https://github.com/tarantool/installer.sh/issues/10
+printf '%s\n' 'Package: tarantool-dev'             | sudo tee    /etc/apt/preferences.d/tarantool-dev
+printf '%s\n' 'Pin: origin download.tarantool.org' | sudo tee -a /etc/apt/preferences.d/tarantool-dev
+printf '%s\n' 'Pin-Priority: 1001'                 | sudo tee -a /etc/apt/preferences.d/tarantool-dev
```

Comment:
I hope we don't need the workaround anymore, because the problem was fixed in installer.sh.

Reply:
Updated.